### ISSUE TITLE
trigsimp should simplify complex exponentials

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy#11459]
```
In [16]: cos(x).rewrite(exp)
Out[16]:
 ⅈ⋅x    -ⅈ⋅x
ℯ      ℯ
──── + ─────
 2       2

In [17]: trigsimp(cos(x).rewrite(exp))
Out[17]:
 ⅈ⋅x    -ⅈ⋅x
ℯ      ℯ
──── + ─────
 2       2
```

**Comment by kritkaran94** [comment]
@asmeurer I would like to work on this issue. How to start ?

**Comment by asmeurer** [comment]
Take a look at the fu module. That would be the best place to add this.

**Comment by kritkaran94** [comment]
@asmeurer What's the expected output for this particular example ?

**Comment by asmeurer** [comment]
It should return the trig function, cos(x).

**Comment by Yathartha22** [comment]
@asmeurer I would like to take this issue.

**Comment by asmeurer** [comment]
I didn't know about that function. Is it new since this issue was made?

**Comment by ylemkimon** [comment]
@asmeurer No, it seems it was commited on 17-19 May 2013(https://github.com/sympy/sympy/commit/7ee3a1e46914bd4bce985e128a29a8962b93bfd8 https://github.com/sympy/sympy/commit/5c22627053be00cde437cb7f2990ab9248032c11) and merged on 22 May 2013(PR #2117).

**Comment by asmeurer** [comment]
It should at least be mentioned in the docstring of trigsimp in the see also section. Or maybe trigsimp should just be calling it (I'm unsure).

**Comment by ylemkimon** [comment]
I think we should integrate `exptrigsimp` into `trigsimp` or change the order of them in `simplify`. `trigsimp` or at least `simplify` should be able to simplify similar forms, but it currently isn't: #8758.

**Comment by czgdp1807** [comment]
`exptrigsimp` works in this case. What should we do? Should we go for the suggestion in https://github.com/sympy/sympy/issues/11459#issuecomment-336678957 or just let the things the way they are right now.

**Comment by czgdp1807** [comment]
@asmeurer Any views on the above comment?
